pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[tool.poetry]
+name = "ngboost"
+version = "0.3.8dev"
+description = "Library for probabilistic predictions via gradient boosting."
+authors = ["Stanford ML Group <avati@cs.stanford.edu>"]
+readme = "README.md"
+homepage = "https://github.com/stanfordmlgroup/ngboost"
+repository = "https://github.com/stanfordmlgroup/ngboost"
+documentation = "https://stanfordmlgroup.github.io/ngboost/intro"
+classifiers = [
+    "Operating System :: OS Independent"
+]
+license = "Apache License 2.0"
+
+[tool.poetry.dependencies]
+python = ">=3.6, <4.0"
+scikit-learn = ">=0.21, <0.24"
+numpy = ">=1.17, <2.0"
+scipy = ">=1.3, <2.0"
+tqdm = ">=4.3, <5.0"
+lifelines = ">=0.25, <0.29"
```

Comment:
note that I've also upper-pinned dependencies here to prevent breaking changes, as-is good software engineering best practices, but this means that as the library is developed, we should keep an eye and update dependencies bounds (and ensure tests still pass) as and when we go about doing it.

Reply:
Yeah for sure